### ISSUE TITLE
CLI: Add validation for column limit

### ISSUE THREAD

**Issue by jgambarios** [dotCMS/core#29084]
### Parent Issue

https://github.com/dotCMS/core/issues/28880

### Task

There is a validation made in the UI that is not done in the CLI. Adding more than four columns to a row should result in an error.

### Proposed Objective

Same as Parent Issue

### Proposed Priority

Same as Parent Issue

### Acceptance Criteria

The CLI should throw an error when more than four columns are added to a row, similar to the UI validation.

>This row is full
The maximum number of columns per row is limited to four.

**Comment by jgambarios** [comment]
**Note to QA:**

This ticket is part of the spike https://github.com/dotCMS/core/issues/28880, and at the end, all the tickets there can be tested as one.

The idea is to ensure that layout fields (such as rows and columns) can be used without issues when using the content type resource. The changes in the content type resource apply some automatic “fixes” to the fields. For example:

1. If no layout fields are sent, a basic layout is created for the content type.
2. If there are more than the maximum allowed columns under a row, the layout is automatically fixed by removing the extra columns and moving the fields accordingly.

**Comment by dcolina** [comment]
Internal QA: Passed.

The functionality works as expected. 

1. We started by defining a new content-type via UI which have been defined with 4 columns (named ContentTest).

2. Then, we have pulled the content through `dotCLI` tool.

3. Then, we have modified the ContentTest.json file (manually), we added an extra column (now ContentTest.json has 5 columns).

4. Finally, we pushed the content through `dotCLI`.

Now we can notice the extra column has been removed.

![Image](https://github.com/user-attachments/assets/5e7f658e-8631-4431-ac5e-bb8304875ae6)

I have attached a content-type file which exceeded the limit of columns allowed.

[ContentTest.json](https://github.com/user-attachments/files/16508767/ContentTest.json)
